### PR TITLE
[11.0][MIG] invoice_comment_template

### PR DESCRIPTION
**Pull request by xavierjimenez**
Migration of the module **invoice_comment_template** to version 11.0.

Depends on:

- [x] https://github.com/OCA/account-invoice-reporting/pull/80 base_comment_template

### CODE REVIEW

File: invoice_comment_template/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+- before invoice lines
+- after invoice lines
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/94/10.0
```
Comment (bodedra) [inline invoice_comment_template/README.rst]: change Odoo version from 10.0 to 11.0

Comment (pedrobaeza) [reply invoice_comment_template/README.rst]: Or better, use the new README by fragments system and regenerate automatically the template.

Comment (xavierjimenez) [reply invoice_comment_template/README.rst]: Yeah, I'll change it to the new README since I always forget about that!

**Review by HaraldPanten — APPROVED** [review]
Functionally review :+1:

**Comment by pedrobaeza** [comment]
Please rebase now that the dependency is merged and rename it to `account_invoice_comment_template` for following the conventions.

**Review by pedrobaeza — APPROVED** [review]
I have finished the migration fixing the tests, the README and renaming translations.

**Comment by xavierjimenez** [comment]
Thanks Pedro!

**Pull request merged**